ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-results-attempt.yml

```diff
@@ -72,6 +72,10 @@ spec:
         if [ -f $RESULTS_FILE ]; then
           echo "Results exist"
           CHECK_RESULTS=$(cat $RESULTS_FILE | jq -r '.passed')
+          if [ $CHECK_RESULTS = "false" ]; then
+            echo "Not all preflight tests passed."
+          fi
+          echo "All preflight tests passed successfully."
```

Comment:
Shouldn't this be inside an else condition?

Reply:
Doesn't this work as else condition?